Microsoft.Toolkit.Mvvm/Microsoft.Toolkit.Mvvm.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <Title>Windows Community Toolkit Mvvm .NET Standard</Title>
+    <Description>
+      This package includes Mvvm .NET Standard code only helpers such as:
+      - ObservableObject: a base class for objects implementing the INotifyPropertyChanged interface.
+      - ViewModelBase: a base class for view-models with built-in minimal dependency injection support.
+      - RelayCommand: a simple delegate command implementing the ICommand interface.
+      - Messenger: a messaging system to exchange messages through different loosely-coupled objects.
+      - Ioc: a helper class to configure dependency injection service containers.
+    </Description>
+    <PackageTags>UWP Toolkit Windows Mvvm observable Ioc dependency injection services extensions helpers</PackageTags>
+  </PropertyGroup>
+
+  <!-- .NET Standard 2.0 doesn't have the Span<T> type -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="System.Memory" Version="4.5.4" />
+  </ItemGroup>
+
+  <!-- .NET Standard 2.1 doesn't have the Unsafe type -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' ">
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.7.1" />
```

Comment:
It doesn't, but it's indirectly getting that reference already from the `System.Memory` package.
This is the same setup we have in `Microsoft.Toolkit` and the `HighPerformance` package - since .NET Standard 2.1 already has the `Span<T>` types, it doesn't need the `System.Memory` reference, so we need to just add the `Unsafe` class manually there. Whereas on .NET Standard 2.0 we just add `System.Memory` which includes everything 👍